### PR TITLE
fix: Rename `getFrameProcessorPlugin` to `initFrameProcessorPlugin`

### PR DESCRIPTION
**Pull request by mrousavy**
## What

cc @mateusz1913

### CODE REVIEW

File: docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_IOS.mdx [hunk 1]
```diff
@@ -71,7 +71,7 @@ VISION_EXPORT_FRAME_PROCESSOR(FaceDetectorFrameProcessorPlugin, detectFaces)
 ```
 
 :::note
-The Frame Processor Plugin will be exposed to JS through the `VisionCameraProxy` object. In this case, it would be `VisionCameraProxy.getFrameProcessorPlugin("detectFaces")`.
+The Frame Processor Plugin will be exposed to JS through the `VisionCameraProxy` object. In this case, it would be `VisionCameraProxy.initFrameProcessorPlugin("detectFaces")`.
 :::
 
 4. **Implement your Frame Processing.** See the [Example Plugin (Objective-C)](https://github.com/mrousavy/react-native-vision-camera/blob/main/package/example/ios/Frame%20Processor%20Plugins/Example%20Plugin%20%28Objective%2DC%29) for reference.
```
Comment (mateusz1913) [inline docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_IOS.mdx]: FYI, those links to examples may not work (I observed link to Swift FP resultingin 404)

**Comment by mateusz1913** [comment]
👍  I added it to https://github.com/mateusz1913/vision-camera-plugin-builder/pull/57 and will release it once you bump VisionCamera

File: docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_IOS.mdx (hunk 1 above)
Comment (mrousavy) [reply docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_IOS.mdx]: thanks, updated!

**Comment by mrousavy** [comment]
@mateusz1913 wdyt of that? https://github.com/mrousavy/react-native-vision-camera/issues/2036#issuecomment-1770403626 Maybe we can register the plugins in that static block, probably better since they're not bound to lifecycle anyways.

**Pull request merged**